contracts/truefi2/BorrowingMutex.sol

```diff
@@ -43,11 +43,11 @@ contract BorrowingMutex is IBorrowingMutex, UpgradeableClaimable {
     function unlock(address borrower) external override {
         address _locker = locker[borrower];
         require(_locker == msg.sender, "BorrowingMutex: Only locker can unlock");
-        locker[borrower] = address(0);
+        locker[borrower] = address(uint256(Status.Unlocked));
```

Comment:
I don't think IBorrowingMutex is a good place for this. We can keep default address const in this file and Blocked address in TCA as it's not strictly connected to the mutex logic.

Reply:
Good point, it's better not to expose implementation details in the interface like this.

I moved both Unlocked and Banned into the BorrowingMutex impl as constants and added a `ban()` function to the interface.